docs/command_line_tool/phylum_auth.md

```diff
@@ -9,5 +9,6 @@ Manage authentication, registration, and API keys
 ### Commands
 * [phylum auth login](https://docs.phylum.io/docs/phylum_auth_login)
 * [phylum auth register](https://docs.phylum.io/docs/phylum_auth_register)
+* [phylum auth set-token](https://docs.phylum.io/docs/phylum_auth_set-token)
```

Comment:
```suggestion
* [phylum auth set-token](https://docs.phylum.io/docs/phylum_auth_set_token)
```
The other possibility is to change the file name to `docs/command_line_tool/phylum_auth_set-token.md`...which would match the other command/page with a hyphenated name: `docs/command_line_tool/phylum_project_set-thresholds.md`

The working assumption here is the file name is what creates the page URL/slug.